src/app/sections/privacy-section.tsx

```diff
@@ -0,0 +1,35 @@
+import home from "@/content/home.json";
+import Image from "next/image";
+import Link from "next/link";
+
+export default function PrivacySection() {
+  const { privacy } = home;
+  return (
+    <section className="bg-[#ecf5ff] p-16 max-w-full mx-auto border-white border-[80px] rounded-lg">
+      <div className="flex flex-col items-center justify-center py-12 px-6 md:p-16 max-w-4xl m-auto">
+        <Image
+          src={privacy.image}
+          alt="Privacy Icon"
+          width={150}
+          height={150}
+          className="mb-6"
+        />
+
+        <h2 className="font-medium text-2xl md:text-3xl text-center text-[#1a1a1a]">
```

Comment:
- font size not big enough
- this text color is already defined in `branch.css`


```suggestion
        <h2 className="font-medium text-4xl md:text-3xl text-center text-navyblue">
```